### PR TITLE
chore(cactus-plugin-ledger-connector-corda): remove unused openapi.yaml

### LINKED ISSUE #3200: chore(cactus-plugin-ledger-connector-corda): remove unused openapi.yaml
### Description

After working on updating the OpenAPI extension there was one openapi.yaml on plugin-ledger-connector-corda that still has x-hyperledger-cactus. When I tried to update it manually, Peter mentioned that it should automatically update after running the generate-server script on package.json. So after further investigation I have found that this openapi.yaml is already irrelevant to the plugin-ledger-connector-corda. I have verified this since we added "documentationProvider: none" on packages/cactus-plugin-ledger-connector-corda/src/main-server/openapi-generator-config.yaml the openapi.yaml does not get updated if we run the generate-server script. As you can also see the changes on this issue #2662 one of the changes is openapi.yaml is removed from the packages/cactus-plugin-ledger-connector-corda/src/main-server/kotlin/gen/kotlin-spring/.openapi-generator/FILES since it is not being used since documentationProvider: none is now implemented.

### PR DESCRIPTION
**Pull request by aldousalvarez**
### **Commit** to be reviewed
---
chore(cactus-plugin-ledger-connector-corda): remove unused openapi.yaml 
```
Primary Changes
----------------
1. Removed the unused openapi.yaml in plugin-ledger-connector-corda
```

Fixes #3200

**Pull Request Requirements**
- [x] Rebased onto `upstream/main` branch and squashed into single commit to help maintainers review it more efficient and to avoid spaghetti git commit graphs that obfuscate which commit did exactly what change, when and, why.
- [x] Have git sign off at the end of commit message to avoid being marked red. You can add `-s` flag when using `git commit` command. You may refer to this [link](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) for more information.
- [x] Follow the Commit Linting specification. You may refer to this [link](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#specification) for more information. 

**Character Limit**
- [x] Pull Request Title and Commit Subject must not exceed 72 characters (including spaces and special characters).
- [x] Commit Message per line must not exceed 80 characters (including spaces and special characters).

**A Must Read for Beginners**
For rebasing and squashing, here's a [must read guide](https://github.com/servo/servo/wiki/Beginner's-guide-to-rebasing-and-squashing) for beginners.

### CODE REVIEW

**Review by jagpreetsinghsasan — APPROVED** [review]
LGTM

**Review by petermetz — COMMENTED** [review]
LGTM, but won't approve until other orgs also reviewed.

**Comment by petermetz** [comment]
@hyperledger/cacti-maintainers Please review this when you get the chance!

**Pull request merged**
